fineract-savings/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountSummary.java

```diff
@@ -83,6 +87,12 @@ public final class SavingsAccountSummary {
     @Column(name = "interest_posted_till_date")
     private LocalDate interestPostedTillDate;
 
+    @Column(name = "accrued_till_date")
+    protected LocalDate accruedTillDate;
+
+    @Column(name = "total_interest_accrued_derived", scale = 6, precision = 19, nullable = true)
```

Comment:
What about accrued fee and penalty?

Reply:
_The Accrual Fee mainly will be in other PR_